Userland/Applications/ClockSettings/ClockSettingsWidget.cpp

```diff
@@ -7,32 +7,50 @@
 #include "ClockSettingsWidget.h"
 #include <Applications/ClockSettings/ClockSettingsWidgetGML.h>
 #include <LibConfig/Client.h>
+#include <LibGUI/CheckBox.h>
 #include <LibGUI/RadioButton.h>
 #include <LibGUI/TextBox.h>
 
+static String time_format_string(bool twelve_hour, bool with_seconds)
+{
```

Comment:
A couple of project style things here:
* Don't need curly braces around single-line `if` or `else`
* Don't need an `else` branch since the `if` branch just returns early
```c++
if (something)
    return foo;
return bar;
```
* We try to use named `enum`s instead of boolean parameters. This is to avoid things like:
```c++
call_some_function(true, false, true, true);
```
At the call sites - it becomes very unclear what the booleans are doing without reading the function. Instead, if we had some `enum` like:
```c++
enum class HourFormat {
    Hour12,
    Hour24,
}
```
Then it's much clearer at the call site what options are being provided. If that becomes really verbose in this instance, it might make sense to make this a `private` member function like `update_time_format_string` that updates `m_custom_format_input` itself.

Reply:
Got it, I've replaced that function with ``update_time_format_string`` which doesn't need any parameters.